src/plugins/unsplash/unsplash.py

```
@@ -12,10 +12,14 @@ def grab_image(image_url, dimensions, timeout_ms=40000):
-        with Image.open(BytesIO(response.content)) as img:
-            resized = img.resize(dimensions, Image.LANCZOS)
-            # Return a copy since we're in a context manager
-            return resized.copy()
+        img = Image.open(BytesIO(response.content))
+        try:
+            return img.resize(dimensions, Image.LANCZOS)
+        finally:
+            try:
+                img.close()
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)